ddr/ddr.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,7 @@
 
 /* Begin PBXBuildFile section */
 		1E23DB8A237A2C84008B8711 /* TutorialImage.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1E23DB89237A2C84008B8711 /* TutorialImage.swift */; };
+		1E2B5CC923898C68007D029C /* Flashlight.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1E2B5CC823898C68007D029C /* Flashlight.swift */; };
```

Comment:
Might just be me but I'm pretty sure this should actually be 
`1E2B5CC923898C65007D029C`